test/lib/cache.js

```diff
@@ -134,20 +134,21 @@ t.test('cache add pkg only', t => {
   })
 })
 
-t.test('cache add pkg w/ spec modifier', t => {
+t.test('cache add multiple pkgs', t => {
   t.teardown(() => {
     logOutput = []
     tarballStreamSpec = ''
     tarballStreamOpts = {}
   })
 
-  cache.exec(['add', 'mypkg', 'latest'], err => {
```

Comment:
We'll want to keep this existing test so we make sure we didn't remove functionality